### DISCUSSION TITLE
Deployment stuck at queued

### DISCUSSION THREAD

**Discussion by leTranTienPhat** [vercel/community#2159] [Help] [answered]
### Summary

Yesterday when I push my codes to github, I noticed that my vercel deployment stuck at "Queued". i thought it was server error or that kind of things, so I waited. But up until this point, my deployments still stuck at Queued. I tried to redeploy, push new code to github, etc... but it just didn't work

**Comment by toniengelhardt** [comment]
I have the same issue, all of my projects get stuck at queued and there are no builds running. Did you find any solution @leTranTienPhat?

**Comment by aryan-mehta05** [comment, marked as answer]
I'm facing the same issue today, but according to previous discussions, the following website shows a status of **Degraded Performance**.
They provide real-time updates on Vercel's status.

[https://www.vercel-status.com/](https://www.vercel-status.com/)

**Reply by aryan-mehta05** [reply]
***[UPDATE]** Yep, the issue has fixed itself for me!*
